grunt/tasks/jasmine.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
     return {
 
       'src': {
-        src: [], // actual src files are require'd in the *.spec.js files
+        src: [
+          'dist/cartodb.uncompressed.js',
+        ],
```

Comment:
Why was this added? The source files already require what they need so this should not be necessary. The "integration" tests that use the bundle was intended to be located in the "cartodb" bundle defined below.